app/koi-client/src/hook/query/Stock/useQueryStock.tsx

```diff
@@ -27,7 +27,7 @@ const useQueryStock = (stockId: string | undefined, options?: Options) => {
 
   const timeIdx = data?.startedTime
     ? Math.floor(getDateDistance(dayjs(data.startedTime).toDate(), new Date()).minutes / data.fluctuationsInterval)
-    : undefined;
+    : -1;
```

Comment:
```suggestion
    : undefined;
```

!timeIdx는 0이라는 숫자도 포함하기 때문에, 수정하는 것은 적절했습니다.
하지만, -1 이라는 숫자는 어떤 의도인지 알기 어렵습니다.
잘못된 값을 명확하게 표기하기 위해서는 `undefined`가 적절해 보입니다.

Reply:
만약 같은 number 타입을 유지하고 싶어서 그런거라면 상수로 관리하는 것도 좋은 방법 같습니다.
```suggestion
const INVALID_IDX = -1
    :  INVALID_IDX ;
```

Reply:
만약에 정말 -1과 같은 음수를 넣고 싶다면, -1 보다는 `Number.NEGATIVE_INFINITY` 를 사용하는 것이 어떨까요

Reply:
뭔가 음수라서 이기보다는 index가 의도된 값이 아니면 findIndex와 같이 -1로 반환하는 경우이니 그런 것이 아닐까 하는 생각입니다.

저도 확실하게 뭐가 더 나아보인다는 잘 모르겠네요 ㅎㅎ

Reply:
findIndex에서 -1 이 반환되어 잘못된 값임을 확인할 수도 있겠지만,
어떤 경우에서는 index -1이 마지막 index를 지칭하는 경우도 있어서 혼란을 줄 수 있다고 생각하네요.
결론적으로 음수로 표현하는 것은 적절하지 않다고 생각합니다.

Reply:
음 확실히 at 메서드의 경우가 딱 그렇겠네요
감사합니다!!  👍👍

Reply:
오홋 넵 undefined; 로 변경해 뒀습니당!